src/labels/CustomLabels.labels

```diff
@@ -48,6 +48,22 @@
         <shortDescription>Error message for TDTM filtering functionality.</shortDescription>
         <value>Please set Filter Field, or both Filter Field and Filter Value.</value>
     </labels>
+    <labels>
```

Comment:
Fun fact, with the new translations that was just merged in to master, I think we have one additional language that we support (British English) - so I think there will be at least one other translations file as part of this PR.

Reply:
I used this command from a new scratch org but looks like I did not receive BE, not sure if I missed anything?

cci task run retrieve_unpackaged -o package_xml src/package.xml -o path src

Reply:
Added British English!